src/starknetCli.ts

```diff
@@ -193,23 +180,18 @@ export function runStarknetDeployAccount(options: IDeployAccountProps) {
   }
 
   const account = options.account ? `--account ${options.account}` : '';
+  const command = 'starknet deploy_account';
 
-  try {
-    execSync(
-      `${warpVenvPrefix} starknet deploy_account --wallet ${options.wallet} --network ${
-        options.network
-      } ${account} ${options.gateway_url ? `--gateway_url ${options.gateway_url}` : ''} ${
-        options.feeder_gateway_url ? `--feeder_gateway_url ${options.feeder_gateway_url}` : ''
-      } ${options.account_dir ? `--account_dir ${options.account_dir}` : ''} ${
-        options.max_fee ? `--max_fee ${options.max_fee}` : ''
-      }`,
-      {
-        stdio: 'inherit',
-      },
-    );
-  } catch {
-    logError('starknet deploy failed');
-  }
+  execSyncAndLog(
+    `${warpVenvPrefix} ${command} --wallet ${options.wallet} --network ${
+      options.network
+    } ${account} ${options.gateway_url ? `--gateway_url ${options.gateway_url}` : ''} ${
+      options.feeder_gateway_url ? `--feeder_gateway_url ${options.feeder_gateway_url}` : ''
+    } ${options.account_dir ? `--account_dir ${options.account_dir}` : ''} ${
+      options.max_fee ? `--max_fee ${options.max_fee}` : ''
```

Comment:
There is a lot of it so let's make a function 
```optionalArg (name, value) => value ? `${name} ${value}`  : ''```

Reply:
nice